tests/deploy/tests/test_metric_manifest.yaml

```diff
@@ -197,7 +200,7 @@ metadata:
     example.com/annotation: "example-annotation"
 spec:
   selector:
-    app: test-pod
+    app: test-statefulset
```

Comment:
The `kube_endpoint_address` is generated only when there is an actual endpoint mapped to the service. Fixing the link between `test-statefulset` and `test-service`, since the StatefulSet already pointed to Service, just not the other way around.